doc/debian/jitsi-meet/jitsi-meet.example

```diff
@@ -40,8 +40,10 @@ server {
     error_page 404 /static/404.html;
 
     gzip on;
-    gzip_types text/plain text/css application/javascript application/json;
+    gzip_types text/plain text/css application/javascript application/json image/x-icon application/octet-stream;
     gzip_vary on;
+    gzip_proxied no-cache no-store private expired auth;
+    gzip_min_length 1000;
```

Comment:
Why limit this to 1000 bytes or more? language-nl.json for example is 873 bytes but compresses nicely to 579 bytes.

Reply:
I guess we could go with 512? I like powers of 2!

Reply:
Done!